daemon/network.go

```diff
@@ -178,6 +178,10 @@ func (daemon *Daemon) SetupIngress(create clustertypes.NetworkCreateRequest, nod
 		if err := ep.Join(sb, nil); err != nil {
 			logrus.Errorf("Failed joining ingress sandbox to ingress endpoint: %v", err)
 		}
+
+		if err := sb.EnableService(); err != nil {
+			logrus.Errorf("Failed enabling service for ingress sandbox: %v", err)
```

Comment:
@dongluochen Let's not be consistently bad. Please use the logging package correctly.